includes/class-newspack-popups.php

```diff
@@ -149,6 +149,18 @@ public static function register_meta() {
 			]
 		);
 
+		\register_meta(
```

Comment:
What's the reason for the `Manual Only` option? Couldn't a manual shortcode placement always take precedence over normal campaign logic, and avoid burdening the user with a difficult-to-understand new setting?

Reply:
_Accidentally removed the comments by cancelling a review._

> Couldn't a manual shortcode placement always take precedence over normal campaign logic

That's easy for shortcodes in post content, but I couldn't make it work for shortcodes in widgets. Looks like widgets are rendered after post content, so the shortcodes rendered in the post content cannot be made aware of their existence.